packages/core/src/html/NodeProcessor.js

```diff
@@ -315,6 +321,60 @@ class NodeProcessor {
     return node;
   }
 
+  /*
+   * Syntax is (space)#char#(any text)##(space)
+   * If we have #char# without ## or ## without #char# we don't replace
+   * To escape start do \#char#, so it will show up as #char# (escaping also works with any front char)
+   * To escape end do \##, which will show up as ##
+   */
+  preprocessMarkdown(content) {
+    const lines = content.split('\n');
+    let amendedContent = '';
+    for (let i = 0; i < lines.length; i += 1) {
+      const lineArray = lines[i].split(' ');
+      const stack = [];
+      for (let j = 0; j < lineArray.length; j += 1) {
+        // Check for #char#
+        const syntaxStartChecker = lineArray[j].slice(0, 3);
```

Comment:
There's a magic number here, could be difficult for future extension

Reply:
Have pulled it out to the front and left comments explaining!